models/stg_amplitude__event.sql

```diff
@@ -71,6 +72,10 @@ final as (
         version_name,
         _fivetran_synced
     from fields
+
+    where cast({{ dbt.date_trunc('day', 'event_time') }} as date) >= {{ "cast('" ~ var('date_range_start',  '2020-01-01') ~ "' as date)" }} -- filter to records past a specific date
+    and cast({{ dbt.date_trunc('day', 'event_time') }} as date) <= {{ "cast('" ~ var('date_range_end', dbt.dateadd("month", 1, dbt.date_trunc("day", dbt.current_timestamp()))  ) ~ "' as date)" }} -- filter to records before a specific date 
```

Comment:
Hi @fivetran-joemarkiewicz , I'm having trouble getting integration tests to pass and the errors are because of this line. I tried single quoting/double quoting and made sure that I'm not missing any parenthesis. Do you see any formatting issues?

Reply:
Hey @fivetran-reneeli first, I think you may need to use `dbt.current_timestamp_backcompat` instead of the `dbt.current_timestamp` since the dbt Labs team raised some concerns around that field being compatible with various versions of dbt-core.

For the issue you are having. Have you been able to see what the compiled output of this line is when you run it locally from the integration_tests directory?